src/inspector/view/qml/MuseScore/Inspector/InspectorForm.qml

```diff
@@ -44,6 +44,20 @@ Rectangle {
         }
     }
 
+    QtObject {
+        id: prv
+
+        property var openedPopup: null
+
+        function closePreviousOpenedPopup(newOpenedPopup) {
+            if (Boolean(openedPopup) && openedPopup !== newOpenedPopup) {
+                openedPopup.close()
+            }
+
+            openedPopup = newOpenedPopup
+        }
+    }
```

Comment:
Why is a `QtObject` needed here? Can't the variable and the function be declared directly?

Reply:
The property declaration approach in QtObject is an imitation of private properties in qml(the engine does not provide such a possibility).